posthog/models/insight.py

```diff
@@ -33,6 +33,7 @@ class Insight(models.Model):
     description: models.CharField = models.CharField(max_length=400, null=True, blank=True)
     team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
     filters: models.JSONField = models.JSONField(default=dict)
+    filter_groups: models.JSONField = models.JSONField(null=True, blank=True)
```

Comment:
Also, RE: migrations. After the changes i just made, it seems like we could just add a simple migration that adds `filter_groups` like this. Insights using new filter pattern can be saved and old pattern still works. If you save a new insight with new filter pattern as long as the original `filters` field is empty, it will parse the new group and use that as the filter